15-lhcb-dev.md

```diff
@@ -55,37 +55,33 @@ Your new development environment won't be very useful without any software to mo
 So why not check out out one of the existing LHCb packages, which are stored in the LHCb SVN repository?
 
 Let's assume you have already used `lb-dev` to set up your development environment and you are currently inside it.
-In order to obtain the source code of the package you want to work on, use `getpack`.
-This is an LHCb-aware wrapper around SVN.
+In order to obtain the source code of the package you want to work on, use the [Git4LHCb](https://twiki.cern.ch/twiki/bin/view/LHCb/Git4LHCb) scripts.
+These are a set of aliases, starting with `git lb-`, that are designed to make developing LHCb software easier.
 For example, if you want to write a custom stripping selection, execute the following in the `DaVinciDev` directory:
 
 ```bash
-getpack Phys/StrippingSelections head
+git lb-use Stripping
+git lb-checkout Stripping/master Phys/StrippingSelections
+make configure
 ```
 
-Under the hood, `getpack` will `svn checkout` (≈ `git clone`) the corresponding SVN repository.
-The first argument to `getpack` is the name of the package you want to checkout, while the second argument allows you to choose a specific branch.
-`head` is usually the one one that contains the newest development changes and the one you should commit new changes to.
+Under the hood, `git lb-use` will add the [`Stripping`](https://gitlab.cern.ch/lhcb/Stripping) repository as a remote in git.
+`git lb-checkout` will then perform a partial checkout of the master branch of the Stripping repository, only adding the files under [`Phys/StrippingSelections`](https://gitlab.cern.ch/lhcb/Stripping/tree/master/Phys/StrippingSelections).
 
-You can now modify the `StrippingSelections` package and run `make` to build it with your changes.
+You can now modify the `StrippingSelections` package and run `make purge && make` to build it with your changes.
```

Comment:
I took this from the [twiki page](https://twiki.cern.ch/twiki/bin/view/LHCb/TestStrippingLine?rev=9#Getpack_the_head_version_of_Phys), I don't know if it's ever actually needed.